cvat-sdk/cvat_sdk/core/proxies/organizations.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2022 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+from cvat_sdk.api_client import apis, models
+from cvat_sdk.core.proxies.model_proxy import (
+    ModelCreateMixin,
+    ModelDeleteMixin,
+    ModelListMixin,
+    ModelRetrieveMixin,
+    ModelUpdateMixin,
+    build_model_bases,
+)
+
+_OrganizationEntityBase, _OrganizationRepoBase = build_model_bases(
+    models.OrganizationRead, apis.OrganizationsApi, api_member_name="organizations_api"
+)
+
+
+class Organization(
+    models.IOrganizationRead,
+    _OrganizationEntityBase,
+    ModelUpdateMixin[models.IPatchedOrganizationWriteRequest],
+    ModelDeleteMixin,
+):
+    _model_partial_update_arg = "patched_organization_write_request"
+
+
+class OrganizationsRepo(
+    _OrganizationRepoBase,
+    ModelCreateMixin[Organization, models.IOrganizationWriteRequest],
+    ModelListMixin[Organization],
+    ModelRetrieveMixin[Organization],
+):
+    _entity_type = Organization
```

Comment:
Probably it will be reasonable to add invitations and memberships as well, because it's quite hard to work with orgs without this. I think, just adding the bases with no extra methods is enough for now.

Reply:
Yeah, fair point. I'll take a look.

Reply:
Turns out, dealing with invitations is tricky, because they're identified by key instead of ID, which breaks some assumptions in the model mixins. I'll try to solve it, but maybe we can postpone that until the next PR?

Reply:
Ok, sounds reasonable.